src/org/zaproxy/zap/extension/anticsrf/AntiCsrfDetectScanner.java

```diff
@@ -86,4 +87,20 @@ public void setLevel(AlertThreshold level) {
 		// Ignore
 	}
 
+	private boolean tagHistoryType(int historyType) {
```

Comment:
True, this was originally hard coded for all rules ;)
The tags only make sense for records that appear in the History tab .. but I was thinking of adding support for displaying other record types in the History tab :/ 
But yeah, good suggestion, will look into it